### ISSUE TITLE
Test failing: tests/cli_test.py::test_print_ics_command

### ISSUE THREAD

**Issue by tlc28** [pimutils/khal#947]
Hi,

On Guix (https://guix.gnu.org/), this test is failing. We have khal v0.10.1 from PyPI, and click v5.1. Click version is important because it (might be) is the cause of the failure.

```
pytest --disable-pytest-warnings tests/cli_test.py -k test_print_ics_command                                                                                                                                                       
============================================================================================================= test session starts =============================================================================================================
platform linux -- Python 3.8.2, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /tmp/khal-0.10.1
plugins: hypothesis-5.4.1, cov-2.8.1
collected 39 items / 38 deselected / 1 selected                                                                                                                                                                                               

tests/cli_test.py F                                                                                                                                                                                                                     [100%]

================================================================================================================== FAILURES ===================================================================================================================
___________________________________________________________________________________________________________ test_print_ics_command ____________________________________________________________________________________________________________

runner = <tests.cli_test.CustomCliRunner object at 0x7f31d61b49a0>

    def test_print_ics_command(runner):
        runner = runner()
        # Input is empty and loading from stdin
        result = runner.invoke(main_khal, ['printics', '-'])
        assert result.exception
    
        # Non existing file
        result = runner.invoke(main_khal, ['printics', 'nonexisting_file'])
        assert result.exception
>       assert (
            'Error: Invalid value for "ics": Could not open file: ' in result.output or
            'Error: Invalid value for "[ICS]": Could not open file:'
            in result.output)
E       assert ('Error: Invalid value for "ics": Could not open file: ' in "Usage: cli printics [OPTIONS] [ICS]\nTry 'cli printics --help' for help.\n\nError: Invalid value for '[ICS]': Could not open file: nonexisting_file: No such file or directory\n" or 'Error: Invalid value for "[ICS]": Could not open file:' in "Usage: cli printics [OPTIONS] [ICS]\nTry 'cli printics --help' for help.\n\nError: Invalid value for '[ICS]': Could not open file: nonexisting_file: No such file or directory\n")
E        +  where "Usage: cli printics [OPTIONS] [ICS]\nTry 'cli printics --help' for help.\n\nError: Invalid value for '[ICS]': Could not open file: nonexisting_file: No such file or directory\n" = <Result SystemExit(2)>.output
E        +  and   "Usage: cli printics [OPTIONS] [ICS]\nTry 'cli printics --help' for help.\n\nError: Invalid value for '[ICS]': Could not open file: nonexisting_file: No such file or directory\n" = <Result SystemExit(2)>.output

tests/cli_test.py:668: AssertionError
```

The error is that the test is looking for 'Invalid value for "[ICS]":'  when the output contains "Invalid value for '[ICS]':". This error message comes from click. I've patched our package definition, but I don't know the proper way to fix this in khal.

**Comment by d7415** [comment]
I think you're reading the error backwards, but this is fixed in master in de06c89652510d7538d22fd9e2da2a44bf1a564c. This is not part of a release yet.

**Comment by tlc28** [comment]
Great! Thanks!
